report/data_tasks/report/create_from_scratch/00_schema/04_create_fdw_schema_lms.jinja2.sql

```diff
@@ -45,7 +41,7 @@ CREATE TYPE report.annotation_sub_type AS ENUM (
         group_bubbled_counts,
         group_map,
         group_roles,
-        organization,
+        organizations,
```

Comment:
This is interesting. This wasn't here at all, but apparently FDW doesn't care. You can replace this with "bibble" and the query runs fine.